moveit_kinematics/ikfast_kinematics_plugin/scripts/create_ikfast_moveit_plugin.py

```diff
@@ -7,9 +7,12 @@
 kinematics service, from within the moveit planning environment, or in
 your own ROS node.
 
-Author: Dave Coleman, CU Boulder
+Author: Michael Lautman PickNik Robotics
+        Dave Coleman, CU Boulder
```

Comment:
demoting me to second because you refactored the functions, eh?

please update my affiliation

Reply:
This was almost a complete rewrite so it felt it was appropriate. Affiliation updated